File: Arcade/Intro/32_absoluteValuesSumMinimization.js
function absoluteValuesSumMinimization(a) {
    var low  = Math.floor((a.length - 1) / 2),
        high = Math.ceil((a.length - 1) / 2);
    var lowTest  = 0,
        highTest = 0;
    for (let i in a) {
        lowTest  += Math.abs(a[i] - a[low]);
        highTest += Math.abs(a[i] - a[high]);
    }
    return highTest >= lowTest ? a[low] : a[high];
}

File: Arcade/Intro/C++/32_absoluteValuesSumMinimization.cpp
int absoluteValuesSumMinimization(std::vector<int> a) {
     int smallestA=0;
     int smallestSum=INT_MAX;
    for(int i=0;i<a.size();i++){
        int sum=0;
   
    
        for(int j=0;j<a.size();j++){
            sum+=abs(a[j]-a[i]);
        }
        if(sum<smallestSum){
            smallestSum=sum;
            smallestA=a[i];
        }
    }
    return smallestA;
}

